src/modules/github/components/pull-list-item-status/pull-list-item-status.test.tsx

```diff
@@ -0,0 +1,141 @@
+import { TooltipProvider } from '@radix-ui/react-tooltip'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
```

Comment:
import these from `@/utils/root/test/testing-library` not `@testing-library/react`. Then you don't need to wrap the component with `TooltipProvider`

Reply:
Ah, so that was the issue… Oh, it was that import causing the problem…
I couldn’t figure out why it wasn’t working without TooltipProvider.”